src/boot/pack.rs

```diff
@@ -37,23 +37,35 @@ fn load_manifest(path: &Path, jump: &Jump) -> Result<(Lift, PathBuf), String> {
     Ok((lift, manifest_path))
 }
 
-#[cfg(target_family = "windows")]
+#[cfg(windows)]
 fn finalize_executable(path: &Path) -> Result<PathBuf, String> {
-    if path.extension().is_none() {
-        let exe = path.with_extension(env::consts::EXE_EXTENSION);
+    use std::ffi::OsStr;
```

Comment:
This was a related bug fix discovered getting the node example IT working on Windows. Previously the `node.js` binary in that example was created as `node.js` on Windows when it should have been `node.js.exe`, which it now is and which is now tested.